.babelrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
## 🔴 Your Pull Request Summary
![image](https://user-images.githubusercontent.com/93039359/231862101-d0336376-be81-4926-b035-438ec4d94668.png)

## 🟢 Descriptive Pull Request Summary
![image](https://user-images.githubusercontent.com/93039359/231862145-8b462668-0882-4868-bc4b-b399a3a19866.png)

- [ ] It would be nice if you make your pull request summary descriptive enough to make your pull request professional and descriptive 👍